client/apps/shipping-settings/view-wrapper.js

```diff
@@ -66,34 +66,42 @@ class LabelSettingsWrapper extends Component {
 	render() {
 		const { carrier, carrierAccounts, liveRatesTypes, subscriptions, isSaving, translate } = this.props;
 
+		// eslint-disable-next-line no-undef
+		const isWCShippingActive = wcConnectData.is_wcshipping_active;
+
 		if ( ! carrier ) {
 			return (
 				<div>
-					<GlobalNotices id="notices" notices={ notices.list } />
-					<MigratorSettings />
-					<LabelSettings onChange={ this.onChange } />
-					<Packages onChange={ this.onChange } />
-					<LiveRatesCarriersList carrierIds={ liveRatesTypes } />
-					<CarrierAccounts accounts={ carrierAccounts } />
-					<SubscriptionsUsage subscriptions={ subscriptions } />
+					<GlobalNotices id="notices" notices={ notices.list }/>
+					<MigratorSettings/>
+					{ isWCShippingActive && ( <div>
+						<h2>
+							<a href="admin.php?page=wc-settings&tab=shipping&section=woocommerce-shipping-settings">{ translate( 'WooCommerce Shipping' ) }</a> &gt; { translate( 'Live rates' ) }
+						</h2>
+					</div> ) }
+					{ !isWCShippingActive && <LabelSettings onChange={ this.onChange }/> }
+					<Packages onChange={ this.onChange }/>
+					<LiveRatesCarriersList carrierIds={ liveRatesTypes }/>
+					{ !isWCShippingActive && <CarrierAccounts accounts={ carrierAccounts }/> }
+					{ !isWCShippingActive && <SubscriptionsUsage subscriptions={ subscriptions }/> }
```

Comment:
Hide sections that are not relevant if WCShipping is active (i.e. only display sections related to live rates).